test/ssl_data/generate.sh

```diff
@@ -0,0 +1,68 @@
+#!/bin/bash
+#
+# This script generates SSL keys and certificates used for testing.
```

Comment:
Please, add a comment with an openssl utility version from which the script could be launched. It may help in the future.
```suggestion
# This script generates SSL keys and certificates used for testing.
```